rollup/types.go

```diff
@@ -278,8 +278,9 @@ func (a *appendSequencerBatchCallData) Decode(r io.ReaderAt) error {
 type CTCTransactionType uint8
 
 const (
-	CTCTransactionTypeEIP155 CTCTransactionType = 0
-	CTCTransactionTypeEOA    CTCTransactionType = 1
+	CTCTransactionTypeEIP155  CTCTransactionType = 0
```

Comment:
This now matches https://github.com/ethereum-optimism/contracts-v2/blob/c1a9e0fa670e3867e032584568edb60954b04820/contracts/optimistic-ethereum/libraries/codec/Lib_OVMCodec.sol#L36